tensorhive/config.py

```diff
@@ -28,17 +35,32 @@ class SSHConfig():
     '''
     AVAILABLE_NODES = {
```

Comment:
I think we should get rid of the specific user credentials in the code, just leave a placeholder like:
AVAILABLE_NODES = {
#'<hostname>': {'<username>': '<password>'}
}
and then maybe detect empty AVAILABLE_NODES in THManager?

Reply:
Fixed credentials: 9b860728fcb0d884fb581f007210b7a36294d611
Fixed empty nodes dict: 154a6ac2df349ee2c584e6decaceb3386fa7e4f5